lib/view/ui/map_page.dart

```diff
@@ -0,0 +1,21 @@
+import 'package:community/utils/font_size_constants.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:community/utils/app_string_res.dart';
+
+class MapPage extends StatelessWidget {
```

Comment:
All classes should be in specific feature folder :
 Example,
lib/view/ui/usermap/map_page.dart